wordpress-mobile/WordPress-iOS
# Jetpack Section: Threat Details File Context

Part of #15190

### Description:
- Added an attributed string for file context in Threat Details

### Merge instructions:
- ⚠️ Merge after https://github.com/wordpress-mobile/WordPress-iOS/pull/15740

### To test:
1. Launch the app, tap on the My Site, tap on a site with Jetpack Scan and also has Threats
2. Tap on a threat cell for a threat that includes a file context
3. ✅ The file context can scroll horizontally
4. ✅ The file context styling should match the design specs  

![Screen Shot 2021-02-01 at 15 48 46](https://user-images.githubusercontent.com/6711616/106424166-0fa84700-64a5-11eb-8299-3be04e13f4b7.png)

### PR submission checklist:
